docs/source/changelog.rst

```diff
@@ -0,0 +1,77 @@
+Changelog
+=========
+
+0.4.0
```

Comment:
There is not even 0.3 - maybe we can remove this section for now ...? 
```suggestion
0.4.0
```

Reply:
There is. Line 25.